lms/djangoapps/courseware/tests/test_access.py

```diff
@@ -862,12 +862,14 @@ def test_course_catalog_access_num_queries(self, user_attr_name, action, course_
         if user_attr_name == 'user_staff' and action == 'see_exists':
             # always checks staff role, and if the course has started, check the duration configuration
             if course_attr_name == 'course_started':
-                num_queries = 2
+                # TODO: course roles: number of querys increased from 2 to 4
```

Comment:
Is this a TODO or more just a notification? If it's a TODO, add a note about what the action is. This comment is true for all places with a TODO that has the query increase, but no action item.

Reply:
It's just to track the original amount of queries in the tests. Thinking about when we will delete the original code, now I think that when we do that the tests are going to fail again, and we are going to have to modify this again. I don't know how useful those comments can be now, do you think it's better to take them out?

Reply:
I agree that once we have fully transitioned to permissions instead of permissions and roles this will change again. I agree the failing tests will alert us at that time.